test_runner/src/main/kotlin/ftl/ios/xctest/common/InstallParseBinaries.kt

```diff
@@ -54,8 +55,16 @@ private fun downloadAndUnzip(osname: String) {
             it.setExecutable(true)
         }
     Files.delete(destinationFile)
-    createSymbolicLinkToFile(
-        link = Paths.get(flankBinariesDirectory.toString(), "libatomic.so.1"),
-        target = Paths.get(flankBinariesDirectory.toString(), "libatomic.so.1.2.0")
-    )
+
+    if (isWindows) {
+        createFileCopy(
+            "$flankBinariesDirectory\\libatomic.so.1.2.0",
+            "$flankBinariesDirectory\\libatomic.so.1"
+        )
+    } else {
+        createSymbolicLinkToFile(
+            link = Paths.get(flankBinariesDirectory.toString(), "libatomic.so.1"),
+            target = Paths.get(flankBinariesDirectory.toString(), "libatomic.so.1.2.0")
+        )
+    }
```

Comment:
I think that we should hide files copy inside `createSymbolicLinkToFile`

Reply:
we should then change createSymbolicLinkToFile to be complient as it will not createSymbolicLinkToFile for windows and therefore the name is not true to its meaning.

I am happy to rename it and do as mentioned but leaving it as createSymbolicLinkToFile is deceptive.

Reply:
yeah, we could think about other name, but using everwhere 
```
if(isWindows) copy else copySymbolicLink 
```

is not good choice IMO

Reply:
I changed the name have a look :)